minh197/WeBlog
# [FEATURE] #4 Fetch text/image for Card component

## Addressing: https://github.com/minh197/WeBlog/issues/4
## How to test: Locally
1. Create a new `.env` file in the `client` folder
2. `REACT_APP_API_KEY=7fbeff6f514f42f59fb852e44770e829`
You can get a new API key from the `https://newsapi.org/` as well (or you can use my API key to test this ticket)
3. Pull the change from my ticket
4. Run `npm start`
5. See that we are able to fetch the data from News API rendered it correctly to our card components
<img width="1485" alt="Screenshot 2024-03-24 at 2 27 44 PM" src="https://github.com/minh197/WeBlog/assets/61608148/354183c3-0f60-4b08-a238-ef1fb9070923">
